src/openforms/js/components/admin/form_design/data/complete-form.js

```diff
@@ -60,6 +60,16 @@ const normalizeLimit = (draft, field) => {
   }
 };
 
+/**
+ * Convert empty str value for a datetime field to null.
+ */
+const normalizeDateTime = (draft, field) => {
+  const form = draft.form;
+  if (form[field] === '') {
+    form[field] = null;
+  }
+};
```

Comment:
Is this specific to a datetime field?

Reply:
As I have implemented this, no. I will fix this and give it a more generic name.